src/bigunit.ts

```diff
@@ -286,15 +286,33 @@ export class BigUnit {
    * @returns BigUnit with the maximum value
    */
   public static max(unit1: BigUnit, unit2: BigUnit): BigUnit {
-    return unit1.gt(unit2) ? unit1 : unit2;
+    return this.compareUnits(unit1, unit2, (u1, u2) => u1.gt(u2));
   }
 
   /**
    * @description Return the minimum of the two values
    * @returns BigUnit with the minimum value
    */
   public static min(unit1: BigUnit, unit2: BigUnit): BigUnit {
-    return unit1.lt(unit2) ? unit1 : unit2;
+    return this.compareUnits(unit1, unit2, (u1, u2) => u1.lt(u2));
+  }
+
+  /**
+   * @description Compares two BigUnit objects based on a comparison function (min or max)
+   * @returns BigUnit based on the comparison function
+   */
+  private static compareUnits(
```

Comment:
I don't think this is useful enough to warrant adding more code to the library, which I would prefer to keep lean and deadly :-)